test/instrumentation/stable_metric_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+
+if diff -u test/instrumentation/testdata/stable-metrics-list.yaml test/instrumentation/stable-metrics-list.yaml; then
+  echo PASS
+  exit 0
+fi
+echo 'See instructions in test/instrumentation/README.md'
```

Comment:
Eventually I think this should probably output something like `Diffs in stable metrics detected, please run "hack/update-stable-metrics.go"` 

And we should eventually also probably have a `hack/update-stable-metrics.go`.

Reply:
Renamed scripts to match name convention with `./hack` scripts. Added `update-stable-metrics.sh` script and allowed ./verify-stable-metrics.sh to be run without bazel.